src/extension/features/accounts/hide-memos/README.md

```diff
@@ -0,0 +1,5 @@
+# Hide the Memo Column
```

Comment:
no need for this readme, we auto-generate features.md when we release :D

Reply:
deleted